src/constants.ts

```diff
@@ -32,21 +37,25 @@ export enum Tokens {
 export const RELAYER_URLS = {
   [Environments.MUMBAI]: "https://mumbai.api.sportx.bet",
   [Environments.PRODUCTION]: "https://app.api.sportx.bet",
+  [Environments.SxToronto]: "https://api.toronto.sx.bet",
 };
 
-export const DEFAULT_MATIC_RPL_URLS = {
+export const DEFAULT_RPL_URLS = {
```

Comment:
Typo here

Reply:
it was there before, so I didn't change the name. fixed